e2e/utils.go

```diff
@@ -481,20 +481,57 @@ func enableTopologyInTemplate(data string) string {
 	return strings.ReplaceAll(data, "--feature-gates=Topology=false", "--feature-gates=Topology=true")
 }
 
-func validatePVCClone(sourcePvcPath, clonePvcPath, clonePvcAppPath string, f *framework.Framework) {
+// nolint:gocyclo // reduce complexity
```

Comment:
this should be `// nolint:gocyclo,gocognit // reduce complexity` ?

Reply:
gocyclo should be enough. I guess the function is too much complex. It will be better to split it.